asset_dashboard/models.py

```diff
@@ -75,26 +75,28 @@ def __str__(self):
 
 
 class ProjectScore(models.Model):
+    min_value = 1
```

Comment:
i think i might think about taking this a step further and define 

`RangeValidators = [MinValueValidator(1), MaxValueValidator(5)]`

and then later.

```
core_mission_score = models.IntegerField(default=1, validators=RangeValidators)
```

Reply:
Neat. Will that allow me to get the class attribute `min_value` or `max_value` so that I can set those in the form rather than hardcoding them? See lines 30 and 31: https://github.com/datamade/ccfp-asset-dashboard/blob/44d696ec196881bfa63a91fafb4fea606d81d9f6/asset_dashboard/forms.py#L30. 

That's the reason I did this, because the model validators weren't making their way into the HTML outputted by the form class. Is there a better way?